development/development.env

```diff
@@ -0,0 +1,42 @@
+################################################################################
+# DEV File: Store environment information. NOTE: Secrets NOT stored here!
+################################################################################
+# Nautobot Configuration Environment Variables
+NAUTOBOT_ALLOWED_HOSTS=*
+NAUTOBOT_BANNER_TOP="Local"
+NAUTOBOT_CHANGELOG_RETENTION=0
+
+NAUTOBOT_DEBUG=True
+NAUTOBOT_DJANGO_EXTENSIONS_ENABLED=True
+NAUTOBOT_DJANGO_TOOLBAR_ENABLED=True
```

Comment:
I'm not familiar with these two flags? I don't see any reference to them in core or in this PR.

Reply:
From the template as well: https://github.com/networktocode-llc/cookiecutter-ntc/blob/nautobot-plugin-v1.0/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/development/development.env#L10

Should I remove these 2 lines?

Reply:
@cmsirbu Seems like these env variables are not used in the `nautobot-plugin` template, should I remove it from there?

Reply:
It's still used in templates to enable Django debug toolbar.

https://github.com/networktocode-llc/cookiecutter-ntc/blob/527dedf686f6ea3cacad773cd7b7e96ee5c1f6ea/nautobot-plugin-ssot/%7B%7B%20cookiecutter.project_slug%20%7D%7D/development/nautobot_config.py#L343

Will align all templates to use the same envs and configuration for this. Keeping it here for now.